apis/python/src/tiledbsoma/common.cc

```diff
@@ -212,6 +212,11 @@ void set_metadata(
     SOMAObject& soma_object, const std::string& key, py::array value) {
     tiledb_datatype_t value_type = np_to_tdb_dtype(value.dtype());
 
+    // For https://github.com/single-cell-data/TileDB-SOMA/pull/2900
```

Comment:
Maybe a short inline summary so future us can be reminded what's going on without clicking links?
```suggestion
    // For https://github.com/single-cell-data/TileDB-SOMA/pull/2900
    // Needed to keep metadata types sync between R and Python APIs.
```